core/src/main/scala/kafka/log/ProducerIdMapping.scala

```diff
@@ -147,7 +145,8 @@ object ProducerIdMapping {
 @nonthreadsafe
 class ProducerIdMapping(val topicPartition: TopicPartition,
                         val config: LogConfig,
-                        val snapParentDir: File) extends Logging {
+                        val snapParentDir: File,
```

Comment:
Ack. Probably thought I would need this initially, but ended up doing that part in a separate PR. I'll remove it here.